helpers/inbound/inbound.go

```diff
@@ -65,7 +70,17 @@ func (email *ParsedEmail) parseRawEmail(rawEmail string) {
 			}
 
 		} else if emailPart.FileName() != "" {
-			email.Attachments[emailPart.FileName()] = readBody(emailPart)
+			headers := make(map[string]string)
+			content := readBody(emailPart)
+
+			headers["Content-Type"] = emailPart.Header.Get("Content-Type")
```

Comment:
Why not just `headers := emailPart.Header` (would need to change type). Alternatively, instead of storing headers, make `ContentType` one of the properties of `ParsedAttachment`, and do some simple string parsing to extract the MIME type from the string returned by `emailPart.Header.Get("Content-Type")`

Reply:
I've changed the ParsedAttachment.Headers to type MIMEHeader and stored the full attachment headers in there. Also added a string ContentType to take the content type (Without the name="***" part). Updated the tests for this too.